src/components/Contact.tsx

```diff
@@ -42,82 +39,75 @@ export default function Contact(props: ContactType) {
+                        PHONE NUMBER
+                       </Text>
+                       <Text mt={{ sm: 3, md: 3, lg: 5 }} color="gray.500" textAlign={"center"}>{props?.phoneNumber}</Text>
+                        </VStack>
                       </Link>
-                    </VStack>
-                  </Box>
-                  <HStack
-                    mt={{ lg: 10, md: 10 }}
-                    spacing={5}
-                    alignItems="flex-start"
-                  >
-                    <IconButton
-                      aria-label="facebook"
-                      variant="ghost"
-                      size="sm"
-                      isRound={true}
-                      bg={'#fff'}
-                      _hover={{ bg: "#0D74FF" }}
-                      icon={<MdFacebook size="16px" />}
-                    />
-                    <IconButton
-                      aria-label="github"
-                      variant="ghost"
-                      size="sm"
-                      isRound={true}
-                      bg={'#fff'}
-                      _hover={{ bg: "#0D74FF" }}
-                      icon={<BsGithub size="16px" />}
-                    />
-                    <IconButton
-                      aria-label="discord"
-                      variant="ghost"
-                      size="sm"
-                      isRound={true}
-                      bg={'#fff'}
-                      _hover={{ bg: "#0D74FF" }}
-                      icon={<BsDiscord size="16px" />}
-                    />
-                  </HStack>
-                </Box>
-              </WrapItem>
-              <WrapItem>
-                <Box bg="white" borderRadius="lg">
-                  <Box m={8} color="#0B0E3F">
-                    <VStack spacing={5}>
+                      </Box>
+                      
+                      <Box bg={"#edf2f7"}><Link href={"mailto:" + props?.email} isExternal display={'fled'} gap={'8px'} alignItems={'center'} w={"10rem"} >
+                        <VStack>
+                           <MdEmail color="#68d391" size="100px" />
+                           <Text mt={{ sm: 3, md: 3, lg: 5 }}  color="gray.500">
+                           EMAIL</Text>
+                           <Text mt={{ sm: 3, md: 3, lg: 5 }} >{props?.email}</Text>
+                          
+                        </VStack>
+                      </Link>
+                    
+                      </Box>
+                      
+
+                      
+                    </SimpleGrid>
+                    </center>
+                    </Box>
+             
+                  
+               
+                <Box boxShadow='md' p='6' rounded='md' bg='white'  borderRadius="lg"  w={"100%"} margin={"5px 0px"}>
+                 
                       <form name="contact" method="post" data-netlify-honeypot="bot-field" data-netlify="true">
+                      <Box py={{ base: 5, sm: 5, md: 8, lg: 10 }}w={"100%"} >
```

Comment:
There appears to be no space between `py `and `w `attribute. Is the `w` still working as expected?